src/widgets/PrivacyWidget.php

```diff
@@ -0,0 +1,175 @@
+    public $forceOutput = false;
+
+    /**
+     * @var string CSS to be applied
+     */
+    public $css = '.privacyPolicyConsent {
+                        position: fixed;
+                        width: 100%;
+                        left: 0;
+                        bottom: 0;
+                        background-color: #A50045;
+                        color: #fff;
+                        z-index: 10000;
+                        display: flex;
+                        padding: 1em;
+                        align-items: center;
+                        justify-content: space-between;
+                    }
+                    .privacyPolicyConsent.top {
+                        top: 0;
+                        bottom: unset;
+                    }
+                    .privacyPolicyConsent a {
+                        color: #fff;
+                    }';
+
+    /**
+     * @var bool Whether to remove css or not
+     */
+    public $removeCSS = false;
+
+    /**
+     * Add a decline cookie. Needed to set the declined privacy policies cookie.
+     */
+    private $addDeclineCookie = false;
+    
+    /**
+     * Translate messages if no widget input
+     */
+    private function setMessages()
+    {
+        if (empty($this->privacyMessage)) {
+            $this->privacyMessage = Module::t('privacy_widget.privacy_message');
+        }
+        if(empty($this->acceptPrivacyButtonText)) {
+            $this->acceptPrivacyButtonText = Module::t('privacy_widget.accept_privacy_button_text');
+        }
+        if(empty($this->declinePrivacyButtonText)) {
+            $this->declinePrivacyButtonText = Module::t('privacy_widget.decline_privacy_button_text');
+        }
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function init()
+    {
+        parent::init();
+        $this->setMessages();
+        $privacyPolicy = Yii::$app->request->post(self::PRIVACY_COOKIE_NAME, null);
+        if (!empty($privacyPolicy)){
+            if ($privacyPolicy === 'true') {
+                Yii::$app->response->cookies->add(new Cookie(["name" => self::PRIVACY_COOKIE_NAME, "value" => true]));
+            } elseif ($privacyPolicy === 'false') {
+                Yii::$app->response->cookies->add(new Cookie(["name" => self::PRIVACY_COOKIE_NAME, "value" => false]));
+                $this->addDeclineCookie = true;
+            }
+        }
+        if (!$this->isPrivacyAccepted()) {
+            Yii::$app->on(Yii::$app::EVENT_AFTER_REQUEST, function ($event) {
+                Yii::$app->response->cookies->removeAll();
+                if ($this->addDeclineCookie) Yii::$app->response->cookies->add(new Cookie(["name" => self::PRIVACY_COOKIE_NAME, "value" => false]));
+            });
+        }
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function run()
+    {
+        if ((!$this->isPrivacyAccepted() && !$this->isPrivacyDeclined()) || $this->forceOutput) {
+            if (!$this->removeCSS) $this->getView()->registerCss($this->css);
```

Comment:
I don't think its a good practice to have $removeCass variable, if people want to remove the defautl css they can just override the css with `'css' => false`